packages/wonder-blocks-popover/components/popover-content-core.js

```diff
@@ -31,7 +36,7 @@ type Props = {|
      * Whether we should use the default light color scheme or switch to a
      * different color scheme.
      */
-    color: "blue" | "dark" | "light",
+    color: "blue" | "darkBlue" | "white",
```

Comment:
Changed colors here: from `dark` to `darkBlue` and from `light` to `white` to match with the tokens we have defined in `wonder-block-colors`. Let me know what you think and if this makes sense, we could start following this pattern in other packages.

Reply:
No change, just a question.

What's the rationale behind using color names instead of semantic names here? Seems like using color names is going to really hamstring theming work.